.buildkite_win/appxmanifest.xml

```diff
@@ -4,10 +4,10 @@
   xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
   xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
   xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10">
-  <Identity Name="EndlessOSFoundation.EndlessKey.app" Version="0.1.10.1" Publisher="CN=EOSF Microsoft Publishing 2022, O=Endless OS Foundation LLC" ProcessorArchitecture="x64" />
```

Comment:
Let's linewrap this so each property is on its own line. We are about to have 3 PRs changing different properties on this line:

- this one
- #44 changing Publisher
- another changing ProcessorArchitecture

Reply:
I've created https://github.com/endlessm/kolibri-app/pull/45 to address that, we can rebase the other PRs once that one gets merged.